plugin/update.go

```diff
@@ -0,0 +1,29 @@
+package plugin
+
+import (
+	"runtime"
+
+	"github.com/turbot/steampipe/constants"
+	"github.com/turbot/steampipe/ociinstaller"
+)
+
+func SkipUpdate(report VersionCheckReport) (bool, string) {
```

Comment:
function comment - explain the algorithm

this function is the core of the logic so make it super clear

Reply:
// 1) If there is an updated version ALWAYS update
	// 2) If we are ARM, current installed version is AMD, and ARM is available
	// 3) Otherwise skip